### ISSUE TITLE
Can't seem to filter loras by name

### ISSUE THREAD

**Issue by VantomPayne** [rgthree/rgthree-comfy#303]
After recent Update All, I can only seem to select loras via a dropdown menu Power Lora Loader comparing to before where I could input text to filter out loras I wish to use. Other loaders such as checkpoint loader and image loader seems to still have the input filter. I apologize if this is caused by another node since I updated all of them.

**Comment by rgthree** [comment]
Hmm.. it seems to be working for me, though I notice it looks like there needs to be a larger number of entries in the immediate drop down. (Looks like it may be 11 or so?)

Are your drop-down lists greater than 11 entries?

**Comment by DrJKL** [comment]
This was broken for me too. I think it's an incompatibility with https://github.com/yolain/ComfyUI-Easy-Use

**Comment by MantisCore** [comment]
For me, it stops working if it gets nested into folders by turning on the "Auto Nest Subdirectories in Menus" option. That is, the filter input still shows up, but nothing is getting filtered. It works fine with this option turned off. I would love for these to work at the same time as I have a huge amount of LoRAs installed and any tool to bring this under control is welcome.

**Comment by AugmentedRealityCat** [comment]
I have the exact same problem as @MantisCore and I also have a huge (**HUGE**) collection of LoRA that just became unusable because it would take me an hour to scroll to the top of the list with my mousewheel !

I haven't tried "Auto Nest Subdirectories in Menus" yet, so that will be my next move. Thanks for sharing that hint, that would be a nice turnaround while we look for a better solution.

**Comment by rgthree** [comment]
For me, if I UNCHECK "Auto Nest Subdirectories in Menus" in the rgthree-comfy settings, I get the default behavior of a single-long list from the Power Lora Load's Add Lora button, AND it had the filter box at the top (since I have more than 11 entries), and typing into that box filter the menu.

If I have CHECKED "Auto Nest Subdirectories in Menus" in the rgthree-comfy settings, I get the rgthree-comfy feature of nested menues which, for me, is an initial list w/o a filter box (since it had less than 11), but the next menu does have a filter box since it has more than 11 entries, and typing into that filter filters the list down as expected. This also works in the next levels of the nested menus as well.

Is it possible there's another extension that's conflicting?

**Comment by AugmentedRealityCat** [comment]
I'll look into conflicting extensions - that's probably the case if it still works for you. It has always worked for me up to maybe a week ago, and I did install many custom nodes since.

**Comment by AugmentedRealityCat** [comment]
I removed all the custom nodes and only kept rgthree and manager to make this test.

If I have CHECKED "Auto Nest Subdirectories in Menus" in the rgthree-comfy settings, I get the rgthree-comfy feature of nested menues which, for me, is an initial list w/o a filter box (since it had less than 11), but the next menu does NOT have a filter box either, even though it has more than 20 entries. I can't type anything.

If I try to do the same but with more than 20 entries in the initial list, then I do get a filter box, but it only ever applies to the items in that initial list, and I can't get that filter box to appear on any sub-folder.

TLDR: filter box only accessible on the initial entry list

**Comment by AugmentedRealityCat** [comment]
Is there anything we can do to help resolve this problem ? Could it be related to the browser I'm using (Firefox 129.0.2) ? Should I try installing previous versions of rgthree or something else and see from which version it stopped working ? Is there a maximum number of LoRA over which this thing would stop working ? I have thousands and thousands of them - hence the need for filtering that long list ! 

Anything else you can think of that I should be checking or reporting back ?

Thanks for your support :)

**Comment by DrJKL** [comment]
You can use the comfy cli to bisect custom nodes.
What custom nodes do you have installed?

**Comment by AugmentedRealityCat** [comment]
I removed ALL the custom nodes and only kept RGthree and Manager to make a test, and it still would not allow me to filter sub-directories.

Any idea about what my next test should be ?

**Comment by DrJKL** [comment]
I'd check the browser console for errors, and if you're up for it maybe add a breakpoint in the menu auto nest code to see if it's getting clobbered somehow.
Your comfy and the comfy frontend are both up to date?

**Comment by AugmentedRealityCat** [comment]
Thanks a lot for the recommendation to look into the browser's console - for some reason I never thought about looking there. There are actually many error messages there, so I'll make some tests and see what I can find.

And yes my comfy is up to date. In fact, We could even say I'm even a bit in advance because I installed pytorch 2.4 + cuda 1.2.4 (which really helps with Flux LoRA training), but the problem I have predates that, so it's unlikely to be related.

**Comment by gobshyte** [comment]
Ok I've just had exactly the same situation, filtering stopped working in sub folders, and it was definitely caused by a custom node conflict of some kind. I only got as far as narrowing it down to these though:

![image](https://github.com/user-attachments/assets/a6c8de9a-e1ff-4bbf-844b-b27d9e7b34f6)
